src/data_types/mod.rs

```diff
@@ -11,6 +11,32 @@ use core::{ffi::c_void, ptr::NonNull};
 #[repr(transparent)]
 pub struct Handle(NonNull<c_void>);
 
+impl Handle {
+    /// Creates a new [`Handle`] from a raw address. The address might
+    /// come from the Multiboot2 information structure or something similar.
+    ///
+    /// # Example
+    /// ```rust
+    /// use core::ffi::c_void;
+    /// use uefi::Handle;
+    ///
+    /// let image_handle_addr = 0xdeadbeef as *mut c_void;
+    ///
+    /// let uefi_image_handle = unsafe {
+    ///     Handle::from_ptr(image_handle_addr).expect("Pointer must not be null!")
+    /// };
+    /// ```
+    ///
+    /// # Safety
+    /// This function is unsafe, because the caller must be sure, that the pointer
```

Comment:
Just a minor nitpick: in English, unlike in Deutsch, it's [not grammatically correct](https://jakubmarian.com/comma-before-that-and-which/) to add commas before subordinate phrases :)

(i.e. there should be no `,`s here)

Reply:
Thanks @GabrielMajeri ! I'm currently writing my master's thesis in English and this is a problem that keeps bothering me 😂